tox.ini

```diff
@@ -74,8 +74,13 @@ deps =
 commands =
     charm: mypy {[vars]src_path} {posargs}
     lib: mypy --python-version 3.5 {[vars]lib_path} {posargs}
+      /usr/bin/env sh -c 'for m in $(git diff --name-only {[vars]lib_path}); do \
+      if ! git diff $m | grep -q "+LIBPATCH"; then \
+        echo "You forgot to bump LIBPATCH on $m!"; exit 1; \
+      fi; done'
```

Comment:
- Indentation seems off
- Would it make sense to move `allowlist_externals` directly above `commands`?
- We need a composite of "LIBAPI.LIBPATCH", hot just LIBPATCH
- Is there a way we could be [more sophisticated](https://stackoverflow.com/questions/1337320/how-to-grep-git-commit-diffs-or-contents-for-a-certain-word) with git usage to avoid the looping etc?

Reply:
* Indenting a tab more in next commit! ✅ 
* `allowlist_externals` is at the very end in all our other environments, I'm just following the style of the file!
* Checking `+LIBPATCH\|+LIBAPI` now ✅ 
* That example would search for **one** `LIBPATCH` being updated across all the changes; if the charm owns two libraries and you only bump one, you wouldn't know through `git log`